packages/saltcorn-data/models/config.ts

```diff
@@ -235,6 +235,68 @@ const configTypes: ConfigTypes = {
     default: "80",
     required: true,
   },
+  min_password_length: {
```

Comment:
have these been implemented? are we checking passwords against these rules?

Reply:
Looks like I had these implemennted in this commit: "[Testing - custom password tuning(Users and security)](https://github.com/saltcorn/saltcorn/pull/3364/commits/da73e3a63acc1d897fddf381a5d421734b0f062f)"

I might have commented and removed them with the new additions